### ISSUE TITLE
Process doesn't handle a SIGKILL (ctrl-c)

### ISSUE THREAD

**Issue by gillespiecd** [pesos/rshark#20]
**Description**

Process doesn't handle a SIGKILL (ctrl-c) to quit, which is expected behavior in a terminal application.

**How To Reproduce**

Run the program, try to quit with ctrl+c, nothing happens.

**Expected Behavior**

 ctrl+c should quit the process

**Desktop**

 - OS: Mac OS
 - Rust version: rustc 1.45.2 (d3fb005a3 2020-07-31)

**Comment by pmk21** [comment]
A fairly straightforward solution would be to add `| Key::Ctrl('c')` here -
https://github.com/pesos/rshark/blob/7cd479911caaafc22f5441b928fb842986b8cc01/src/display/ui.rs#L150

But, I think that this should be handled using a separate signal handler thread. 
If you have another solution in mind, feel free to open a PR and we can discuss it there.

**Comment by gillespiecd** [comment]
>I think that this should be handled using a separate signal handler thread.

I agree, there are also crates that can handle this. It would be the correct approach versus hardcoding the key.

**Comment by pmk21** [comment]
Would you like to take this up @gillespiecd?
